src/IOUService/web/index.html

```diff
@@ -40,21 +40,19 @@ <h3 class="col-md-12">Send a <em>thank you</em> note!</h3>
             </div>
             
             <h3>Notes sent to me</h3>
-            <button type="button" class="btn btn-primary">
+            <button type="button" class="btn btn-primary" id="refresh_button">
```

Comment:
Name: refresh_button: maybe we can find a better name which conveys the purpose of the button more clearly? refresh_received_notes_buttton? or something along these lines